Source/WebCore/platform/graphics/mac/controls/InnerSpinButtonMac.h

```diff
@@ -33,14 +33,14 @@ namespace WebCore {
 
 class InnerSpinButtonPart;
 
-class InnerSpinButtonMac : public ControlMac {
+class InnerSpinButtonMac final : public ControlMac {
 public:
     InnerSpinButtonMac(InnerSpinButtonPart&, ControlFactoryMac&);
 
 private:
-    IntSize cellSize(NSControlSize, const ControlStyle&) const override;
+    IntSize cellSize(NSControlSize, const ControlStyle&) const final;
```

Comment:
I believe if the class is `final`, putting `final` on functions rather than `override` does nothing.